vite-base.config.ts

```diff
@@ -61,8 +61,12 @@ export default defineConfig({
   server: {
     port: 8080
   },
+  /**
+   * Some old libs like `promise-queue` and `readable-stream` still uses Webpack.
+   */
   define: {
-    'process.env': {} // some old libs like `promise-queue` still uses Webpack
+    'process.browser': {},
```

Comment:
Most probably a boolean value expected. Note: It's stringified on purpose

```suggestion
    'process.browser': 'true',
```

Reply:
https://github.com/Adamant-im/adamant-im/pull/800/commits/03bf622fa219b6e02ba0919f3e08c869d94b9eaf